### PR TITLE
Fix for rotation

### PR DESCRIPTION
**Pull request by DenysMoro**

### CODE REVIEW

**Review by aviklai — CHANGES_REQUESTED** [review]
@DenysMoro I think we should keep package-lock.json:
https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5

**Comment by DenysMoro** [comment]
@aviklai reverted back

File: examples/screens/Images/Images.tsx [hunk 1]
```diff
@@ -13,11 +13,21 @@ const imageAfter = require('../../assets/images/after.jpg');
 type ScreenImagesProps = StackScreenProps<never, Screens.IMAGES>;
 
 export const ScreenImages: React.FC<ScreenImagesProps> = () => {
+  const { width, height } = useWindowDimensions();
+  const [size, setSize] = useState({ width, height: width / 2 });
```
Comment (dinataranis) [inline examples/screens/Images/Images.tsx:17]: @DenysMoro can you please explain why do we use `height: width / 2` ? 
I think we need to do something like `{width: 100%, height: 'auto'}` (responsive resizing)

**Review by aviklai — APPROVED** [review]

**Review by antonazaides — APPROVED** [review]

File: examples/screens/Images/Images.tsx (hunk 1 above)
Comment (aviklai) [reply examples/screens/Images/Images.tsx:17]: @DenysMoro ?

**Pull request merged**